packages/browser/src/Widget/Server/Server.js

```diff
@@ -10,9 +10,13 @@ export { GeocodingService } from './Geocoding/services/GeocodingService';
 
 export { GeocodingView } from './Geocoding/views/GeocodingView';
 
+export { SparqlEndpointResponseProvider } from './SPARQL/Service/SparqlEndpointResponseProvider';
+
 export { SparqlQueryWindow } from './SPARQL/View/SparqlQueryWindow';
 
-export { SparqlEndpointResponseProvider } from './SPARQL/Service/SparqlEndpointResponseProvider';
+export { SparqlWorkspaceQueryWindow } from './Workspace/View/SparqlWorkspaceQueryWindow';
+
+export * as URI from './SPARQL/Model/URI';
```

Comment:
if this file is part of the SPARQL folder it should be the same in the API

Reply:
By api do you mean move the workspace window from
`./Workspace/View/SparqlWorkspaceQueryWindow`
to
`./SPARQL/Workspace/View/SparqlWorkspaceQueryWindow`?